src/components/ui/Loader/Loader.tsx

```diff
@@ -0,0 +1,26 @@
+import { memo } from 'react';
```

Comment:
No tests for Loader

Reply:
Loader is just calling the library. Do we need to test something that has been tested by the library already?

Reply:
Just our code, so test if classname is applied and it's enough

Reply:
Sure no problem